presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -87,7 +111,58 @@ protected String toSqlType(Type type)
         if (VARBINARY.equals(type)) {
             return "bytea";
         }
+        if (JSON.equals(type)) {
+            return "jsonb";
+        }
 
         return super.toSqlType(type);
     }
+
+    @Override
+    public Optional<ReadMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
+    {
+        if (typeHandle.getJdbcType() == Types.OTHER) {
+            switch (typeHandle.getJdbcTypeName()) {
+                case "jsonb":
+                case "json":
+                    return Optional.of(jsonReadMapping());
+            }
+        }
+
+        return super.toPrestoType(session, typeHandle);
+    }
+
+    private static ReadMapping jsonReadMapping()
+    {
+        return sliceReadMapping(JSON, (resultSet, columnIndex) -> jsonParse(utf8Slice(resultSet.getString(columnIndex))));
+    }
```

Comment:
@findepi 
Any idea how can I use ```com.facebook.presto.operator.scalar.JsonFunctions#jsonParse``` from here without the need to duplicate code ?
Same for ```com.facebook.presto.type.JsonType``` that is needed by ```ReadMapping```